### PR TITLE
Update Creodias API

### LINKED ISSUE #9: Exchanging EOLab Finder with EOLab Data Explorer
The EOLab Finder will no longer be maintained from May 11, 2024 onwards. As a result, we will be moving to the EOLab Data Explorer.

### PR DESCRIPTION
**Pull request by jsreuss**
Closes #9

### CODE REVIEW

File: eurocropsml/acquisition/collector.py [hunk 1]
```diff
@@ -361,41 +363,39 @@ def _process_batch(args: tuple[int, int, gpd.GeoDataFrame, gpd.GeoDataFrame, Pat
...
 ) -> list | None:
     """Getting information from raster .SAFE-files."""
-    safe_file: str = tile["properties"]["productIdentifier"]
+    safe_file: str = tile["S3Path"]  # product Identifier
+    cloudcover = _get_dict_value_by_name(tile["Attributes"], "cloudCover")
+    endingdate = _get_dict_value_by_name(tile["Attributes"], "endingDateTime")
 
     if eodata_dir is not None:
         safe_file = safe_file.replace("eodata", eodata_dir)
-    if f"{processing_level}_N0500" not in safe_file:
```
Comment (jsreuss) [inline eurocropsml/acquisition/collector.py:373]: These are now already filtered out within the API request (see line 51-57)

File: eurocropsml/acquisition/collector.py [hunk 2]
```diff
@@ -47,21 +48,23 @@ def _eolab_finder(
...
-        year,
         months_list[1],
         num_days,
-        max_cloud_cover,
         processing_level[1:],
         geom_polygon,
         products_per_request,
     )
-
+    print(request_url)
```
Comment (jmaces) [inline eurocropsml/acquisition/collector.py:67]: Is the print intended as user feedback here? Then we should rely on the `logger.info(...)` functionality instead.

If it was just for debugging purposes we should remove it before merging.

**Review by jmaces — COMMENTED** [review]
Code changes look good. Could you check if this also requires modifications to the documentation that reflect the change made (e.g. mentions of EOLab Finder)?

File: eurocropsml/acquisition/collector.py (hunk 2 above)
Comment (jsreuss) [reply eurocropsml/acquisition/collector.py]: Sorry, that was still from debugging. I removed it :)

**Comment by jmaces** [comment]
> Code changes look good. Could you check if this also requires modifications to the documentation that reflect the change made (e.g. mentions of EOLab Finder)?

Done. Docs were updated in commit 5c9128f

**Review by jmaces — APPROVED** [review]

**Pull request merged**